.stylelintignore

```diff
@@ -1,4 +1,9 @@
 packages/design-tokens/**/*.scss
 packages/design-tokens/**/*.css
 packages/components/locales/*
-storybook/styles/*
+packages/components/dist/*
+packages/design-tokens/dist/*
+packages/tailwind/dist/*
```

Comment:
I wonder if just `dist` would work 🤔 But no need to change

Reply:
It maybe kinda didn't work, but only components needs to be here cause the others don't contain CSS

Reply:
Haha drat! Whatever works :)